src/main/java/io/github/moulberry/notenoughupdates/profileviewer/DungeonPage.java

```diff
@@ -371,38 +394,54 @@ public void drawPage(int mouseX, int mouseY, float partialTicks) {
 
 				getInstance().tooltipToDisplay =
 					Lists.newArrayList(
-						EnumChatFormatting.YELLOW + "Remaining XP: " + EnumChatFormatting.GRAY + String.format("%,d", floorLevelToXP),
+						EnumChatFormatting.YELLOW + "Remaining XP: " + EnumChatFormatting.GRAY +
+							String.format("%,d", floorLevelToXP),
 						String.format("# M3 Runs (%s xp) : %d", StringUtils.shortNumberFormat(xpM3), runsM3),
 						String.format("# M4 Runs (%s xp) : %d", StringUtils.shortNumberFormat(xpM4), runsM4),
 						String.format("# M5 Runs (%s xp) : %d", StringUtils.shortNumberFormat(xpM5), runsM5),
 						String.format("# M6 Runs (%s xp) : %d", StringUtils.shortNumberFormat(xpM6), runsM6),
-            String.format("# M7 Runs (%s xp) : %d", StringUtils.shortNumberFormat(xpM7), runsM7),
+						String.format("# M7 Runs (%s xp) : %d", StringUtils.shortNumberFormat(xpM7), runsM7),
 						""
 					);
 				boolean hasTime = false;
 				if (timeM3 > 1000) {
 					getInstance()
-						.tooltipToDisplay.add(String.format("Expected Time (M3) : %s", Utils.prettyTime(runsM3 * (long) (timeM3 * 1.2))));
+						.tooltipToDisplay.add(String.format(
```

Comment:
Code duplication AAAAAAAAH

Reply:
Still no need to change this in this pr isnt related at all

Reply:
This PR adds lots of new lines at this spot that are not necessary at all. Just because its easier to just don't change existing code at all, you can't just discard my concerns about unnecessary code duplication..

Reply:
I have no intention in changing your concerns about code duplication. Efe just wont do it because there is no reason for him to do it. This has nothing to do with this pull request